### PR TITLE
Adding sandbox environment to available configuration option and updating the address to the staging URL

### PR DESCRIPTION
**Pull request by metade**
Hi,

I've updated the library as it seems that Just Giving have created a new sandbox environment and changed the URL for the staging environment:
https://api.justgiving.com/docs/usage#environments

I had to fix faraday_middleware back to 0.7 as it looks like it's API has changed - not sure if there's a neater way to handle this...

Cheers,
Patrick

### CODE REVIEW

File: lib/just_giving.rb [hunk 1]
```diff
@@ -9,4 +9,4 @@
 require 'just_giving/fundraising'
 require 'just_giving/search'
 require 'just_giving/donation'
-require 'just_giving/railtie' if defined?(Rails)
\ No newline at end of file
+require 'just_giving/railtie' if defined?(Rails) && Rails::VERSION::MAJOR >= 3
```
Comment (bob-p) [inline lib/just_giving.rb:12]: Do we need to check the version here?

Comment (metade) [inline lib/just_giving.rb:12]: Ideally yes - it allows the gem to be used in Rails 2.

**Comment by metade** [comment]
Thanks for merging the changes.
Could you please release a new version of the gem?

**Comment by bob-p** [comment]
Bumped the version

**Comment by metade** [comment]
Thanks!
It doesn't seem to have released a new version of the gem on Rubygems though?
http://rubygems.org/gems/just_giving/versions

**Pull request merged**